data/human/culture conversations.txt

```diff
@@ -103,6 +103,35 @@ mission "Trash Fire Crops"
 
 
 
+mission "Worker's Occasion"
+	minor
+	source
+		government "Republic"
+		attributes "dirt belt"
+		not attributes "station"
+	to offer
+		random < 1
+	on offer
+		conversation
+			`Pausing to consider a restaurant for your dinner, you notice a local worker staring at the menu, hand shaking as he proceeds to walk in. His clothes are neatly worn, and look freshly brushed, but it's impossible not to notice the dirty marks and stains that just haven't washed out. The man is undeniably nervous and twitchy, yet no one kicks him out. Looking at the menu yourself, you realize the prices are very affordable for you but you recall how unreachable they would've been for you before your career change.`
+			choice
+				`	(Offer to buy the man's dinner.)`
+					goto aid
+				`	(Pity the man.)`
+					goto pity
+				`	(Smile at the man.)`
+			`	You smile at the man and reflect on the value of small achievements. He catches your smile out of the corner of his eye and gives you a hesitant but friendly wave back before you go back about your business.`
+				decline
+			label aid
+			`	You enter the restaurant and catch the man's attention, offering to buy his meal.`
+			`	He looks at your pilot's attire and frowns immediately before telling you where you can stick that offer in colorful terms. On reflection you think that perhaps your pride would've been rather heavily wounded were you in his position too.`
```

Comment:
"your pilot's attire" makes it sound like he's talking about the attire of your pilot to me.

Reply:
hmmmmmm

Reply:
```suggestion
			`	He recognizes you as a pilot and frowns, before telling you where you can stick that offer in colorful terms. On reflection you think that perhaps your pride would've been rather heavily wounded were you in his position too.`
```

```suggestion
			`	He looks looks you over for a second, before frowning and telling you where you can stick that offer in colorful terms. On reflection you think that perhaps your pride would've been rather heavily wounded were you in his position too.`
```

Scowl would be another choice, though it doesn't help with the problem at hand.

Reply:
```suggestion
			`	He recognizes you as a pilot and frowns, before telling you where you can stick that offer in colorful terms. On reflection, you think that perhaps your pride would've been rather heavily wounded were you in his position.`
```